src/coreanimation.cs

```diff
@@ -178,10 +178,10 @@ interface CADisplayLink {
 		[Export ("paused")]
 		bool Paused { [Bind ("isPaused")] get; set; }
 
-		[Deprecated (PlatformName.iOS, 10, 0, message: "Use 'PreferredFramesPerSecond' property.")]
-		[Deprecated (PlatformName.TvOS, 10, 0, message: "Use 'PreferredFramesPerSecond' property.")]
-		[Deprecated (PlatformName.WatchOS, 3, 0, message: "Use 'PreferredFramesPerSecond' property.")]
-		[Deprecated (PlatformName.MacCatalyst, 13, 1, message: "Use 'PreferredFramesPerSecond' property.")]
+		[Obsoleted (PlatformName.iOS, 10, 0, message: "Use 'PreferredFramesPerSecond' property.")]
+		[Obsoleted (PlatformName.TvOS, 10, 0, message: "Use 'PreferredFramesPerSecond' property.")]
+		[Obsoleted (PlatformName.WatchOS, 3, 0, message: "Use 'PreferredFramesPerSecond' property.")]
+		[Obsoleted (PlatformName.MacCatalyst, 13, 1, message: "Use 'PreferredFramesPerSecond' property.")]
```

Comment:
Not obsoleted in macOS?

Reply:
Should have a NoMac, Since the new method should be used and has no point to add it.